system-variables.md

```diff
@@ -283,6 +283,18 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - When this value is greater than `0`, TiDB will batch commit statements such as `INSERT` or `LOAD DATA` into smaller transactions. This reduces memory usage and helps ensure that the `txn-total-size-limit` is not reached by bulk modifications.
 - Only the value `0` provides ACID compliance. Setting this to any other value will break the atomicity and isolation guarantees of TiDB.
 
+### `tidb_enable_amend_pessimistic_txn` <span class="version-mark">New in v4.0.7</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: 0
+- This variable is used to control whether to enable the `AMEND TRANSACTION` feature. If you enable the `AMEND TRANSACTION` feature with a pessimistic transaction, when there are concurrent DDL operations and SCHEMA VERSION changes on tables associated with this transaction, TiDB attempts to amend the transaction. It corrects the commit of the transaction to make it consistent with the latest valid SCHEMA VERSION, so that the transaction can be successfully committed without getting the `Information schema is changed` error. This feature is effective for the following concurrent DDL operations:
```

Comment:
```suggestion
- This variable is used to control whether to enable the `AMEND TRANSACTION` feature. If you enable the `AMEND TRANSACTION` feature in a pessimistic transaction, when concurrent DDL operations and SCHEMA VERSION changes exist on tables associated with this transaction, TiDB attempts to amend the transaction. TiDB corrects the transaction commit to make the commit consistent with the latest valid SCHEMA VERSION so that the transaction can be successfully committed without getting the `Information schema is changed` error. This feature is effective on the following concurrent DDL operations:
```